mmdet3d/models/detectors/centerpoint_two_stage.py

```diff
@@ -0,0 +1,168 @@
+# Copyright (c) OpenMMLab. All rights reserved.
```

Comment:
We may need to adjust the names of functions and variables to be more consistent with one-stage CenterPoint, and that  will make it possible to develop multi-modality methods based on this model.

Reply:
Besides, do you think which detector would be better for this model inherit from? TwoStage3DDetector or CenterPoint

Reply:
> 

I will try it. I'm also interested in multi-modality. Keep interface consistent is necessary.

Reply:
> Besides, do you think which detector would be better for this model inherit from? TwoStage3DDetector or CenterPoint (although I think the current is ok)

Current CenterPoint inherits from `MVXTwoStageDetector` which does not seems to support two-stage points detector. That's why I choose `TwoStage3DDetector`.

Reply:
> We may need to adjust the names of functions and variables to be more consistent with one-stage CenterPoint, and that will make it possible to develop multi-modality methods based on this model.

Do you mean, for example, change `extract_feat()` in `CenterPointTwoStage` to `extract_pts_feat()` like the one-stage `CenterPoint`? If so, the two-stage CenterPoint would better inherit from `MVXTwoStageDetector`. However, `MVXTwoStageDetector` seems to be unable to support two-stage lidar detector. Maybe `MVXTwoStageDetector` should be reimplemented. Otherwise, the first stage head and the second stage head need to be implement in a single `pts_bbox_head`, which is kind of ugly.